drivers/gpio/gpio_nrfx.c

```diff
@@ -67,10 +67,14 @@ static void gpiote_channel_free(u32_t abs_pin)
 	u32_t intenset = nrf_gpiote_int_is_enabled(NRF_GPIOTE_INT_IN_MASK);
 
 	for (size_t i = 0; i < GPIOTE_CH_NUM; i++) {
+		nrf_gpiote_events_t evt =
+			offsetof(NRF_GPIOTE_Type, EVENTS_IN[i]);
```

Comment:
It will work correctly, but I do not know if it is not too much. How about changing `offsetof` to `NRF_GPIOTE_EVENTS_IN_0 + i`?

Reply:
this is copy/paste from isr handler code in the same file